incident.go

```diff
@@ -154,11 +154,12 @@ type CreateIncidentOptions struct {
 
 // ManageIncidentsOptions is the structure used when PUTing updates to incidents to the ManageIncidents func
 type ManageIncidentsOptions struct {
-	ID          string        `json:"id"`
-	Type        string        `json:"type"`
-	Status      string        `json:"status,omitempty"`
-	Priority    *APIReference `json:"priority,omitempty"`
-	Assignments []Assignee    `json:"assignments,omitempty"`
+	ID               string            `json:"id"`
+	Type             string            `json:"type"`
+	Status           string            `json:"status,omitempty"`
+	Priority         *APIReference     `json:"priority,omitempty"`
+	Assignments      []Assignee        `json:"assignments,omitempty"`
+	EscalationPolicy *EscalationPolicy `json:"escalation_policy,omitempty"`
```

Comment:
While I know this is technically an `EscalationPolicy`, looking at the [PagerDuty API documentation](https://developer.pagerduty.com/api-reference/reference/REST/openapiv3.json/paths/~1incidents/put) it seems like this expects the [APIObject](https://pkg.go.dev/github.com/PagerDuty/go-pagerduty#APIObject) subset of the `EscalationPolicy` type.

I'm not sure if the PagerDuty will always return success if we rendered the other `EscalationPolicy` fields in the JSON document. That makes me think we should update this to be an `*APIObject`.

What are your thoughts?